samples/urlshortener/src/main/kotlin/com/squareup/urlshortener/UrlShortenerModule.kt

```diff
@@ -0,0 +1,28 @@
+package com.squareup.urlshortener
+
+import misk.MiskModule
+import misk.config.ConfigModule
+import misk.config.MiskConfig
+import misk.environment.Environment
+import misk.environment.EnvironmentModule
+import misk.hibernate.HibernateEntityModule
+import misk.hibernate.HibernateModule
+import misk.inject.KAbstractModule
+import misk.resources.ResourceLoaderModule
+
+/** Binds dependencies for all environments. */
+class UrlShortenerModule(val environment: Environment) : KAbstractModule() {
+  override fun configure() {
+    val config = MiskConfig.load<UrlShortenerConfig>("urlshortener", environment)
+    install(ConfigModule.create("urlshortener", config))
+
+    install(MiskModule())
+    install(ResourceLoaderModule())
```

Comment:
I wonder if `ResourceLoaderModule` and `EnvironmentModule` should be in `MiskModule`?

Reply:
This needs work. MiskModule is an awkward catch-all.

I think maybe we restructure modules to be task-focused first (ResourceLoaderModule, EnvironmentModule, ServiceManagerModule) and then offer convenient aggregates (MiskServiceModule, MiskTestModule) ? Something smarter than what we have now.

Reply:
sure, I just thought `MiskModule` was the convenient aggregate.

Reply:
Maybe? It might need a new name